srcpkgs/openmsx/template

```diff
@@ -1,17 +1,16 @@
 # Template file for 'openmsx'
 pkgname=openmsx
-version=0.13.0
-revision=2
+version=0.14.0
+revision=1
 short_desc="The MSX emulator that aims for perfection"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-license="GPL-3"
+license="GPL-2.0"
```

Comment:
Checked with the source code tree - the mish-mash of headers seems to imply that the combined work is exclusively GPLv2.

Reply:
should be "GPL-2.0-only" then